subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AbstractTaskRelocationIntegrationTest.groovy

```diff
@@ -18,11 +18,6 @@ package org.gradle.integtests.fixtures
 
 abstract class AbstractTaskRelocationIntegrationTest extends AbstractIntegrationSpec {
 
-    @Override
-    def setupBuildOperationFixture() {
-        //disable because of a test that is incompatible with the build operation fixture
-    }
-
```

Comment:
:question: 
when you don't disable it here,
Shouldn't there be a call to `disableProblemsApiCheck`somewhere?

Reply:
It's not enabled by default (for now).
I think we can defer doing this job after we will turn on the checking by default, which is a bit far into the future.